### DISCUSSION TITLE
Does anyone understand the funkin' rhythm system and can briefly explain it.

### DISCUSSION THREAD

**Discussion by rayanmargham** [FunkinCrew/Funkin#1862] [Q&A]
Title says it. Asking this cause i'm under pressure from other funkin' remakes and i need to get working on the arrow system ASAP.
I fail to understand what the ms timings in the json files for the arrows really mean and how funkin' handles them.

**Comment by steviegt6** [comment]
I would love to help, but if you are constructing your own system then you should attempt to learn and understand these things yourself. Last I checked, the milliseconds are for timing within the song, and hold lengths are for sustain arrow lengths (also measured in milliseconds for their length, I believe). This is all off the top of my head.
